signer/src/bin/demo_cli.rs

```diff
@@ -117,14 +117,17 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
         .expect("No Emily endpoints configured");
     let emily_client_config = Configuration {
         base_path: emily_api_endpoint_config
-            .endpoint
             .to_string()
             .trim_end_matches("/")
             .to_string(),
```

Comment:
Will the `to_string` include the username (ie, the api key)?

Reply:
Yeah it will, we don't have an api key in devenv so it wasn't a problem, but I guess you run this against testnet with keys don't you... will add the same check here

Reply:
I was wondering if that "username" may accidentally get leaked in some logs

Reply:
Nah, it's not an actual username, this is just a URI-format hijack (like `postgres://user:pass@host.port/database`). We take this directly and map it to the generated emily code's `api_key` -- it's only the format that has changed.

Reply:
I think this should do it [2fc4d2e](https://github.com/stacks-network/sbtc/pull/961/commits/2fc4d2e66a098bf13ad1d3be406cf0fa11744999)

Reply:
I think you should port that same change also in `signer/src/emily_client.rs`